File: srm182/ib_evaluator.py
class IBEvaluator:
	def getSummary(self, predictedGrades, actualGrades):
		diff = [0 for x in range(7)]
		for i in range(len(predictedGrades)):
			tmp = abs(predictedGrades[i]- actualGrades[i])
			diff[tmp] += 1
		
		n = float(len(predictedGrades))
		for i in range(len(diff)):
			diff[i] = int(diff[i] / n * 100)
		
		return diff 

File: srm182/IBEvaluator.java
import java.lang.*;

public class IBEvaluator {

	public int[] getSummary(int[] predictedGrades, int[] actualGrades) {
		int[] diff = new int[7];
		int n = predictedGrades.length;
		for (int i = 0; i < diff.length; i++) {
			diff[i] = 0;
		}
		for (int j = 0; j < n; j++) {	
			int tmp = Math.abs(predictedGrades[j]- actualGrades[j]);
			diff[tmp] += 1;
		}
		for (int l = 0; l < diff.length; l++) {
			float tmp = diff[l] / (float)n;
			diff[l] = (int)Math.floor(tmp * 100);
		}
		
	return diff;
	}
} 